src/boards/sparkfun_nrf52840_micromod/board.h

```diff
@@ -59,7 +59,7 @@
 //------------- UF2 -------------//
 #define UF2_PRODUCT_NAME       "SparkFun MicroMod nRF52840"
 #define UF2_VOLUME_LABEL       "SFMM852BOOT"
-#define UF2_BOARD_ID           "micromod-nRF52840"
+#define UF2_BOARD_ID           "nRF52840-micromod-v10"
```

Comment:
you need to make it clear that it is `v1.0` v10 is easily mistaken as version ten. But I am fine with existing code though.

Reply:
I think it *is* version 10 though. I was basing that on the schematic published on the product page, [here](https://cdn.sparkfun.com/assets/f/0/9/9/e/MicroMod_Processor_Board-nRF52840.pdf)  which has "v10" in the `REV` area.

By comparison, their similar ESP32 is [v11](https://cdn.sparkfun.com/assets/2/2/5/9/5/MicroMod_ESP32_Schematic.pdf), and the SAMD51 flavor [v12](https://cdn.sparkfun.com/assets/1/d/3/e/b/SparkFun_MicroMod_SAMD51_Processor_Board_Schematic.pdf). 

But their RP2040 one is [v1.7](https://cdn.sparkfun.com/assets/1/8/3/5/8/MicroMod-RP2040-ProcessorBoard_Schematic.pdf), and the Artemis one is [v1.6](https://cdn.sparkfun.com/assets/3/e/0/b/5/16401_MicroMod-Artemis-ProcessorBoard_schematic.pdf)

In short: Maybe those are supposed to be v1.0, v1.1, and v1.2? Should we assume it's v1.0 in this case or just match what it says on the schematic?

Reply:
really, ok sorry, I normally didn't see that high number on the pcb/board world.